modules/ROOT/pages/to-change-raml-version.adoc

```diff
@@ -1,22 +1,44 @@
-= To Change a RAML Version
+= Change a REST API Version
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
+:imagesdir: ../assets/images/
 
-Changing a RAML version cannot be done in Anypoint Exchange. Instead, you can change the RAML version
-either in the RAML specification itself or when you publish to Anypoint Exchange using Design Center.
+For both RAML and OAS API specifications within Exchange, you can change the version number from within Exchange using the *Add new version* button in the Exchange user interface.
 
-From the RAML specification:
+image::ex-add-new-version.png[]
+
+This opens the Publish a new asset window. 
+
+image::ex-publish-a-new-asset.png[]
```

Comment:
The image is including SNAPSHOT's feature that will not be in production with AMF Release

Reply:
Fixed. I removed the Snapshot checkbox from the image and added it to the branch.